Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/autotunevend.yml

```diff
@@ -0,0 +1,57 @@
+- type: vendingMachineInventory
+  id: AutoTuneVendInventory
+  startingInventory:
+    TrumpetInstrument: 2
+    TromboneInstrument: 2
+    FrenchHornInstrument: 2
+    SaxophoneInstrument: 2
+    EuphoniumInstrument: 2
+    AcousticGuitarInstrument: 2
+    ElectricGuitarInstrument: 2
+    BassGuitarInstrument: 2
+    RockGuitarInstrument: 2
+    BanjoInstrument: 2
+    ViolinInstrument: 2
+    CelloInstrument: 2
+    ViolaInstrument: 2
+    RecorderInstrument: 2
+    BagpipeInstrument: 2
+    ClarinetInstrument: 2
+    FluteInstrument: 2
+    HarmonicaInstrument: 2
+    OcarinaInstrument: 2
+    PanFluteInstrument: 2
+    SynthesizerInstrument: 2
+    AccordionInstrument: 2
+    KalimbaInstrument: 2
+    WoodblockInstrument: 2
+    GlockenspielInstrument: 2
+    BikeHornInstrument: 1
+    MusicBoxInstrument: 1
+    SeashellInstrument: 1
+    XylophoneInstrument: 1
+    GunpetInstrument: 1
+    MicrophoneInstrument: 1
+    HelicopterInstrument: 1
+    BirdToyInstrument: 1
+    MusicalLungInstrument: 1
+    ReverseCymbalsInstrument: 1
```

Comment:
same with reverse cymbals and canned applause, they are straight aweful on the ears

Reply:
Fixed.